tests/functional/adapter/test_basic.py

```diff
@@ -50,3 +58,58 @@ class TestSnapshotTimestampDatabricks(BaseSnapshotTimestamp):
 
 class TestBaseAdapterMethodDatabricks(BaseAdapterMethod):
     pass
+
+
+class TestDocsGenerateDatabricks(BaseDocsGenerate):
+    @pytest.fixture(scope="class")
+    def expected_catalog(self, project):
+        return base_expected_catalog(
+            project,
+            role=AnyString(),
+            id_type="long",
+            text_type="string",
+            time_type="timestamp",
+            view_type="view",
+            table_type="table",
+            model_stats=no_stats(),
+        )
+
+
+class TestDocsGenReferencesDatabricks(BaseDocsGenReferences):
+    @pytest.fixture(scope="class")
+    def model_stats(self, is_uc):
+        if is_uc:
+            return no_stats()
+        else:
+            return {
+                "bytes": {
```

Comment:
I am wondering why do we need "bytes" and "has_stats" here vs no stats for UC?

Reply:
This is an expected value of the stats for `ephemeral_summary` in the test.
Seems like currently non-UC clusters return the stats somehow.